tests.sh

```diff
@@ -163,9 +206,18 @@ function run_cmd_on_client(){
 }
 
 function get_daos_status(){
+    local TEARDOWN_ON_ERROR="${1}"
+    set_default TEARDOWN_ON_ERROR true
+
     run_cmd_on_client "dmg -o ${DAOS_CONTROL_YAML} pool list"
     run_cmd_on_client "dmg -o ${DAOS_CONTROL_YAML} pool query --pool ${POOL_UUID}"
-    run_cmd_on_client "dmg -o ${DAOS_CONTROL_YAML} system query"
+
+    get_server_status ${DAOS_SERVERS} true
+    RC=$?
+    if [ ${TEARDOWN_ON_ERROR} = true ] && [ $RC -ne 0 ]; then
+        pmsg_err "Bad server status"
+        teardown_test
+    fi
```

Comment:
This should help catch some of the "unexpectedly evicted" issues, since we were running dmg system query periodically anyway